api/instancemutater/machine.go

```diff
@@ -201,7 +201,6 @@ func (m *Machine) CharmProfilingInfo() (*ProfileInfo, error) {
 	}
 	profileChanges := make([]ProfileChanges, len(result.ProfileChanges))
 	for i, change := range result.ProfileChanges {
-		profileChanges[i].Subordinate = change.Subordinate
 		if change.Profile != nil {
 			profileChanges[i].Profile = &CharmLXDProfile{
```

Comment:
There's an lxdprofile.Profile{} we can use now instead of the local CharmLXDProfile{}.  If so, can CharmLXDProfiles{} be removed?